libnavigator/src/main/java/com/mapbox/navigation/navigator/MapboxNativeNavigatorImpl.kt

```diff
@@ -101,7 +101,11 @@ object MapboxNativeNavigatorImpl : MapboxNativeNavigator {
 
     // Offline
 
-    override fun configureRouter(routerParams: RouterParams, httpClient: HttpInterface): Long =
+    override fun cacheLastRoute() {
```

Comment:
Should we expose a `NavigationOptions` boolean flag that determines whether we call this method or not after we set the route? We can set it up in a following PR as well.

Reply:
So you mean not exposing this method at all and add a flag to `setRoute` (via `NavigationOptions`) and control if `cacheLastRoute` should be called or not? If so, yeah that's an option, we can also control that upstream in `MapboxTripSession` / `MapboxNavigation` (`MapboxTripSession` doesn't accept `NavigationOptions` but I believe it's going to be needed at some point). Is that what you were thinking? In any case, we can follow up in a separate PR as you mentioned and not block this one 👍

Reply:
One of those works, we can either pass the options or call from upstream. Let's decide in the follow up :+1: